### ISSUE TITLE
♿ a11y(bal-input-slider): focus not visible enough

### ISSUE THREAD

**Issue by zsofia-oroszi-ibmix** [baloise/design-system#1630]
### Description of this issue

Focus is not visible enough.

<img width="296" alt="Image" src="https://github.com/user-attachments/assets/5d2da3de-b6ff-4395-99d0-7d48e01e2a6a" />

### How to fix it

Better to have rectangle around the element like example: https://www.w3.org/WAI/ARIA/apg/patterns/slider/examples/slider-color-viewer/

<img width="422" alt="Image" src="https://github.com/user-attachments/assets/72e6c02b-878e-44a4-9f37-8087688de615" />

### Code Reproduction URL

https://design.baloise.dev/?path=/story/components-form-inputslider--basic&globals=framework:html

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by GinaBiondo** [comment]
The focus is not visible at all when the input is in focus.
